src/main/java/com/cloudbees/jenkins/support/filter/SensitiveContentFilter.java

```diff
@@ -47,37 +50,44 @@
 @Restricted(NoExternalUse.class)
 public class SensitiveContentFilter implements ContentFilter {
 
+    private static final Logger LOGGER = Logger.getLogger(SensitiveContentFilter.class.getName());
+
+    private final transient ThreadLocal<Pattern> mappingsPattern = new ThreadLocal<>();
+    private final transient ThreadLocal<Map<String, String>> replacementsMap = new ThreadLocal<>();
```

Comment:
It is not a persisted configuration. Though it seems to be used over remoting for example at https://github.com/jenkinsci/support-core-plugin/blob/1283.v9ddb_0284a_00c/src/main/java/com/cloudbees/jenkins/support/impl/FileDescriptorLimit.java#L122-L128.

@jglick I am not sure what that would mean then. We should remove transient because of the serialization over remoting ? I have tested with and without and that FileDescriptorLimit component seems to word 🤔

Reply:
Maybe your testing missed some aspect. #474

Reply:
Oh I just realized, all this time I was looking at the file descriptor of the master nodes 🤦 
So yeah apparently this was never working... Can be fixed by either making ContentFilters `Serializable` or `SerializableOnlyOverRemoting`, or doing filtering on the controller side (on the result, but before caching - in `AyncResultCache`).

Reply:
Removed the `transient` as suggested.